jina/clients/base/helper.py

```diff
@@ -171,7 +171,10 @@ async def send_message(self, request: 'Request'):
                 response = await self.session.post(
                     **request_kwargs
                 ).__aenter__()
-                r_str = await response.json()
+                try:
+                    r_str = await response.json()
+                except aiohttp.ContentTypeError:
+                    r_str = response.text
```

Comment:
This should be `response.text()` since we are using aiohttp package.
https://docs.aiohttp.org/en/stable/client_quickstart.html#make-a-request

Reply:
Weird, I tried, and `response.text` was the one that worked here. If this comes up as an error I will create a quick fix.

Reply:
@girishc13 could it be that you have already `awaited` the `response`?

Reply:
Ah yes, could be. Since we have already have the response the `.text` is readily accessible.